lib/cache/repositories/impressions/redis_repository.rb

```diff
@@ -39,14 +39,15 @@ def add_bulk(key, bucketing_key, treatments_labels_change_numbers, time)
           # delete fetched impressions afterwards
           def clear
             impressions = impression_keys.each_with_object([]) do |key, memo|
-              _, _, ip, = key.split('/')
+              ip = key.split('/')[-2] # 'prefix/sdk_lang/ip/impressions.name' -> ip
```

Comment:
`[-2]` asks for the second element, but it count from right to left
`a = [1, 2, 3]`
`a[-2] # => 2`